crates/rpc/rpc-engine-api/src/engine_api.rs

```diff
@@ -44,6 +47,7 @@ const MAX_BLOB_LIMIT: usize = 128;
 /// functions in the Execution layer that are crucial for the consensus process.
 pub struct EngineApi<Provider, EngineT: EngineTypes, Pool, Validator, ChainSpec> {
     inner: Arc<EngineApiInner<Provider, EngineT, Pool, Validator, ChainSpec>>,
+    start_time: Mutex<Option<Instant>>,
```

Comment:
this should go into the inner type, because the `EngineApi` type is cloned per request